ui/plugins/pluginpanel.html

```diff
@@ -206,6 +206,13 @@
           <label>Log parsing status:</label>
           <span>No logs</span>
         </li>
+        <li class="small" ng-click="backfillJob()">
+          <label>
+            <a title="Trigger this jobtype back to the previous time it was run">
+                Backfill this job
+            </a>
+          </label>
+        </li>
```

Comment:
I think we're at the point for creating a small nested drop down menu in the Job details action bar, either for general icon overflow or for new items like this where they don't need to be as prominent. We've an open bug for this but the number escapes me offhand. It's late here but I'll try to dig it up tomorrow.